src/istgt_lu_ctl.c

```diff
@@ -2845,7 +2845,37 @@ _verb_stat SCSIstat_rslt[SCSI_ARYSZ] = { {0,0,0} };
 _verb_istat ISCSIstat_last[ISCSI_ARYSZ] = { {0,0,0} };
 _verb_istat ISCSIstat_now[ISCSI_ARYSZ] = { {0,0,0} };
 _verb_istat ISCSIstat_rslt[ISCSI_ARYSZ] = { {0,0,0} };
-
+static int
+istgt_uctl_cmd_iostats(UCTL_Ptr uctl)
+{
+        const char *delim = ARGS_DELIM;
+        char *arg;
+        ISTGT_LU_Ptr lu;       
+        int rc;
+        int i, j;
+        ISTGT_LU_DISK *spec;
+        ISTGT_Ptr istgt = uctl->istgt;
+        for (i = 0; i < MAX_LOGICAL_UNIT; i++) {
+                lu = (ISTGT_LU_Ptr *) istgt->logical_unit[i];
+                if (lu == NULL)
+                        continue;
+                for (j = 0; j< lu->maxlun; j++) { 
+                	spec = (ISTGT_LU_DISK *) lu->lun[j].spec;
+			if (spec == NULL)
+				continue;
+                	istgt_uctl_snprintf(uctl, "%s  IQN=%s Blockcount=%lu Blocklength=%lu Writes=%lu Reads=%lu TotalReadBytes=%lu TotalWriteBytes=%lu Size=%lu\n", uctl->cmd, lu->name, spec->blockcnt, spec->blocklen, spec->writes, spec->reads, spec->readbytes, spec->writebytes, spec->size);
+                	rc = istgt_uctl_writeline(uctl);
+                }
+                if (rc != UCTL_CMD_OK)
```

Comment:
you should be writing error on the socket

Reply:
ok